vnext/Shared/DevSupportManager.cpp

```diff
@@ -21,6 +21,11 @@
 #include <winrt/Windows.Web.Http.Headers.h>
 #include <winrt/Windows.Web.Http.h>
 
+#ifdef HERMES_ENABLE_DEBUGGER
+#include <winrt/Windows.ApplicationModel.Activation.h>
```

Comment:
it looks like you only use Windows.ApplicationModel below, i.e. no classes in the Activation namespace, is that right? if so, you can remove this include

Reply:
Without this include, the compiler says: error C3779: 'winrt::Windows::ApplicationModel::Package::Current': a function that returns 'auto' cannot be used before it is defined